monai/optimizers/utils.py

```diff
@@ -48,7 +52,7 @@ def generate_param_groups(
         print(net.named_parameters())  # print out all the named parameters to filter out expected items
         params = generate_param_groups(
             network=net,
-            layer_matches=[lambda x: x.model[-1], lambda x: "conv.weight" in x],
+            layer_matches=[lambda x: x.model[0], lambda x: "2.0.conv" in x[0]],
```

Comment:
May I know what `x[0]` is here?

Reply:
yes,`network.named_parameters()` returns a tuple of (name, parameter), the filter should check "2.0.conv" in name, not the tuple...